toolbox/__init__.py

```diff
@@ -237,22 +237,30 @@ def vocoder_progress(i, seq_len, b_size, gen_rate):
         wav = wav / np.abs(wav).max() * 0.97
         self.ui.play(wav, Synthesizer.sample_rate)
 
-        #Enable replay and save buttons:
-        self.ui.replay_wav_button.setDisabled(False)
-        self.ui.export_wav_button.setDisabled(False)
+        # Name it (history displayed in combobox)
+        # TODO better naming for the combobox items?
+        wav_name = str(self.waves_count + 1)
```

Comment:
This is what I meant by adding a placeholder so we can implement a better naming system when we have it figured out. This is preferable to regenerating the entire list every time a wav is updated.